src/lib.rs

```diff
@@ -24,11 +30,9 @@ struct SidecarConfig {
     token: *const c_char,
     schema: *const c_char,
     introspection: *const c_char,
+    egress_url: *const c_char,
 }
 
-#[macro_use]
```

Comment:
what did this do?
could it be the reason lib was needed at compile time?

Reply:
lib is not needed since https://github.com/inigolabs/inigo-rs/commit/01310384d088d3e88f5be18a31da244e3eb4e981

it is needed for lazy loading, see https://docs.rs/lazy_static/latest/lazy_static/#example